config/migrations-triggering-indexing/2025/20251017100323-add-test-municipality.sparql

```diff
@@ -0,0 +1,106 @@
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    <http://data.lblod.info/id/werkingsgebieden/00cfb328-1082-470d-9328-e615fbe42308>
+      a <http://www.w3.org/ns/prov#Location> ;
+      <http://mu.semte.ch/vocabularies/core/uuid> "00cfb328-1082-470d-9328-e615fbe42308" ;
+      <http://www.w3.org/2000/01/rdf-schema#label> "testgemeente" ;
+      <http://mu.semte.ch/vocabularies/ext/werkingsgebiedNiveau> "Gemeente" ;
+      <http://www.w3.org/2004/02/skos/core#exactMatch> <http://vocab.belgif.be/auth/refnis2025/13013> .
+  }
+
+  GRAPH <http://mu.semte.ch/graphs/shared> {
+    <http://data.lblod.info/id/bestuurseenheden/115d9fdc-4858-4fc8-b0ac-033263425222>
+      a <http://www.w3.org/ns/org#Organization> ,
+        <http://data.vlaanderen.be/ns/besluit#Bestuurseenheid> ,
+        <http://purl.org/dc/terms/Agent> ;
+      <http://mu.semte.ch/vocabularies/core/uuid> "115d9fdc-4858-4fc8-b0ac-033263425222" ;
+      <http://www.w3.org/ns/org#classification> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001> ;
+      <http://www.w3.org/2004/02/skos/core#prefLabel> "testgemeente" ;
+      <http://www.w3.org/ns/regorg#legalName> "testgemeente" ;
+      <http://www.w3.org/ns/regorg#orgStatus> <http://lblod.data.gift/concepts/63cc561de9188d64ba5840a42ae8f0d6> .
+
+    <http://data.lblod.info/id/bestuurseenheden/52cc723a-9717-496d-9897-c6774cf124e4>
+      <http://www.w3.org/ns/org#hasSubOrganization> <http://data.lblod.info/id/bestuurseenheden/115d9fdc-4858-4fc8-b0ac-033263425222> .
+  }
+
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+    <http://data.lblod.info/id/bestuurseenheden/115d9fdc-4858-4fc8-b0ac-033263425222>
+      <http://www.w3.org/ns/adms#identifier> <http://data.lblod.info/id/identificatoren/6d60f06a-2eab-4b05-9f2d-138a13a5d1f8> ;
+      <http://www.w3.org/ns/adms#identifier> <http://data.lblod.info/id/identificatoren/cfaf6807-5982-4015-ad3c-276ad72dcce2> ;
+      <http://data.vlaanderen.be/ns/besluit#werkingsgebied> <http://data.lblod.info/id/werkingsgebieden/00cfb328-1082-470d-9328-e615fbe42308> ;
+      <http://www.w3.org/ns/org#hasPrimarySite> <http://data.lblod.info/id/vestigingen/e407c87e-4923-43c9-aa95-3cd6a576c81e> .
+
+    <http://data.lblod.info/id/bestuurseenheden/52cc723a-9717-496d-9897-c6774cf124e4>
+      <http://www.w3.org/ns/org#hasSubOrganization> <http://data.lblod.info/id/bestuurseenheden/115d9fdc-4858-4fc8-b0ac-033263425222> .
```

Comment:
That relation is a bit weird but it's editable in the frontend so I suppose it's fine :)